PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
+
+## Pull Request Etiquette
+
+There are several guidelines you should follow in order for your
+  Pull Request to be merged.
+
+- [ ] I have checked the PRs for upcoming features/bug fixes.
+- [ ] I have contacted the staff regarding bigger changes.
+- [ ] I have read the [contributing][contributing] rules.
+- [ ] I have updated my fork to the latest `upstream/master` changes.
+- [ ] I have provided proper commit captions.
+- [ ] I have included documentation if applicable.
+- [ ] I have made my code consistent with the rest of JDA.
+
+> It is sometimes better to include more changes in a single commit. 
+  If you find yourself having an overwhelming amount of commits, you
+  can **rebase** your branch.
+
+## Pull Request
+
+Provide a general description of what your Pull Request does.
```

Comment:
This should be replaced by the issuer, maybe make that more clear in the sentence